src/envoy/http/authn/jwt_authn_utils.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/envoy/http/authn/jwt_authn_utils.h"
+
+namespace Envoy {
+namespace Http {
+namespace Istio {
+namespace AuthN {
+
+namespace {
+// The name for JWT cluster
+// Todo: need to add a cluster field in the Istio authn JWT config.
+// Before such field is added to the Istio authn JWT config,
+// it is temporarily hard-coded.
+const std::string kJwtClusterName("example_issuer");
+}  // namespace
+
+// Convert istio-authn::jwt to jwt_auth::jwt in protobuf format.
+void convertJwtAuthFormat(
+    const ::istio::authentication::v1alpha1::Jwt& jwt_authn,
+    Http::JwtAuth::Config::AuthFilterConfig* proto_config) {
+  // Todo: when istio-authn::jwt diverges from jwt_auth::jwt,
+  // may need to convert more fields.
+  auto jwt = proto_config->add_jwts();
+  MessageUtil::jsonConvert(jwt_authn, *jwt);
+  jwt->set_jwks_uri_envoy_cluster(kJwtClusterName);
```

Comment:
There should be a function to construct the cluster name from config (e.g from jwks_uri and issuer etc). E.g here is how pilot did it for current jwt config https://github.com/istio/istio/blob/c863c92ff0384fa5f343c6575b4fe9095a249703/pilot/pkg/proxy/envoy/v1/mixer.go#L401

You can make a similar function here.

Reply:
To be clear, this is also for quick prototype. Later, pilot may need to put that information in the config. We'll need introduce new proto for that at some point.

Reply:
Done.